vsts/cleanup.yaml

```diff
@@ -0,0 +1,21 @@
+name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+jobs:
+  - job: WINDOWS
+    displayName: Windows
+    pool:
+      vmImage: windows-2022
+    steps:
+      - powershell: ./vsts/cleanup.ps1
+        displayName: build
+        env:
+          IOTHUB_CONNECTION_STRING: $(IOTHUB-CONNECTION-STRING)
+          PROVISIONING_CONNECTION_STRING: $(PROVISIONING-CONNECTION-STRING)
+          STORAGE_ACCOUNT_CONNECTION_STRING: $(STORAGE-ACCOUNT-CONNECTION-STRING)
+
+      - task: ComponentGovernanceComponentDetection@0
```

Comment:
Does every pipeline require this check? If we could remove this then it would make our maintenance cost even lower. Our cleanup samples wouldn't give us any beneficial component governance results.

Reply:
It should be auto-injected in anyways, so I can delete this block